README.md

```diff
@@ -14,45 +14,31 @@ c2patool is a command line tool for working with C2PA [manifests](https://c2pa.o
 
 ## Installation
 
-### via Homebrew
-
-You can install c2patool on Mac and Linux via [Homebrew](https://brew.sh/):
+If you have [Rust](https://www.rust-lang.org/tools/install) installed, you can install c2patool using:
 
 ```shell
-brew tap contentauth/tools
-brew install c2patool
+cargo install c2patool
 ```
 
-### Building from source
-
-If you have Rust installed, you can build c2patool from source:
+Or you can clone the repo and build:
 
 ```shell
 git clone git@github.com:contentauth/c2patool.git
 cargo build
 ```
 
-## Optional prerequisites
-
-### Manifest definition format
-
-Writing or previewing a manifest requires you to provide a manifest definition, which is a JSON data structure
-that describes the manifest data. The JSON schema for this file is available at [`schemas/manifest-definition.json`](./schemas/manifest-definition.json).
-
-**Note:** Any file references specified will be relative to the location of the manifest definition file.
+Or you can use [Homebrew](https://brew.sh/) on MacOS or Linux to install everything you need:
 
-### Creating certificates
-
-You should be able to test creating your own manifests using pre-built certificates supplied with this tool. However, if
-you want to create your own official certificates, please reference the section titled ["Creating and using an X.509 certificate"](#creating-and-using-an-x509-certificate).
+```shell
+brew tap contentauth/tools
+brew install c2patool
```

Comment:
should this appear above `cargo install` as the preferred method, and should we also indicate that `brew update` is supported? Do we need `cargo install` at all?

Reply:
@andyparsons `cargo install` is cross-platform; `brew` is not. Until we cover more platforms with standardized installations, I think `cargo` should come first.

Reply:
Cargo install should work on Windows where we don't support brew.